config/services.yaml

```diff
@@ -8,13 +8,13 @@ services:
                             # The best practice is to be explicit about your dependencies anyway.
         bind:
             $officialEndpoint: '%env(FLEX_OFFICIAL_ENDPOINT)%'
-            $enableProxy: '%proxy_official_endpoint%'
-            $cacheEndpoint: '%cache_official_endpoint%'
+            $enableProxy: '%env(bool:PROXY_OFFICIAL_ENDPOINT)%'
```

Comment:
Great, thanks for the change! This is not backwards compatible though, so we would have to have new major version in order to release this change. We should be able to have some backwards compatible solution which sets existing parameters as defaults for the env() processors if existing. Are you willing and able to figure out a solution?

Reply:
Sure, how about assigning them to parameters first like it was before? They'll still get their defaults from the `.env` file that way.